shell/utils/__tests__/formatter.test.ts

```diff
@@ -0,0 +1,76 @@
+import { formatEncryptionSecretNames } from '@shell/utils/formatter';
+
+describe('formatter', () => {
+  const secrets = [
```

Comment:
It would be good to include one secret with metadata.state.error === true to test the one other bit of filtering in this function.